app/assets/stylesheets/v2/application.scss

```diff
@@ -0,0 +1,34 @@
+// From GDS's alphagov/govuk_frontend_toolkit
+@import 'colours';
+@import 'font_stack';
+@import 'measurements';
+@import 'conditionals';
+@import 'device-pixels';
+@import 'grid_layout';
+@import 'typography';
+@import 'shims';
+
+@import 'design-patterns/alpha-beta';
+@import 'design-patterns/buttons';
+@import 'design-patterns/breadcrumbs';
+
+// From GDS's alphagov/govuk_elements
+@import 'elements/helpers';
+@import 'elements/reset';
+
+
+@import 'elements/layout';
+@import 'elements/elements-typography';
+@import 'elements/buttons';
+// @import 'elements/icons';
+@import 'elements/lists';
+@import 'elements/tables';
+@import 'elements/details';
+@import 'elements/panels';
+@import 'elements/forms';
+// @import 'elements/forms/form-block-labels';
+@import 'elements/forms/form-date';
+@import 'elements/forms/form-validation';
+@import 'elements/breadcrumbs';
+@import 'elements/phase-banner';
+@import 'elements/components';
```

Comment:
**[nit pick/not a blocker]** missing EOL.

💭 Consider [setting your editor of choice to set this by default](https://atom.io/packages/eol-enforcer).

Reply:
this good to go?